staging/src/k8s.io/client-go/tools/clientcmd/loader.go

```diff
@@ -528,27 +527,6 @@ func RelativizeAuthInfoLocalPaths(authInfo *clientcmdapi.AuthInfo) error {
 	return nil
 }
 
-func RelativizeConfigPaths(config *clientcmdapi.Config, base string) error {
```

Comment:
These ought to stay.  Also, if no one is using them, then something is probably going wrong.